src/server/requirements.txt

```diff
@@ -1,7 +1,7 @@
 flask~=2.2.3
-gunicorn~=20.0.4
+gunicorn==20.1.0
 pandas~=1.5.3
-pyopenssl~=23.0.0
+pyopenssl==23.1.0
```

Comment:
Why == and not ~=? I am aware that those two are most recent releases, but keep in mind that new releases will come in the future and we will most probably want to use those

Reply:
PyCharm's doing 😅 I'll fix it